app/scripts/metamask-controller.js

```diff
@@ -514,6 +529,14 @@ module.exports = class MetamaskController extends EventEmitter {
 
       // onboarding controller
       setSeedPhraseBackedUp: nodeify(onboardingController.setSeedPhraseBackedUp, onboardingController),
+
+      // 3Box
+      setThreeBoxSyncingPermission: nodeify(threeBoxController.setThreeBoxSyncingPermission, threeBoxController),
+      restoreFromThreeBox: nodeify(threeBoxController.restoreFromThreeBox, threeBoxController),
+      setRestoredFromThreeBox: nodeify(threeBoxController.setRestoredFromThreeBox, threeBoxController),
```

Comment:
This is a bit of a nitpick, but maybe this method should be changed to no longer accept a `bool` parameter. It is never and should never be called with a `true` parameter, because that happens as part of `restoreFromThreeBox`.

Maybe it could be called `setNotRestoredFromThreeBox` - sounds kinda dumb, but I can't think of a better sounding suggestion at the moment.

Reply:
Addressed in 37f6dfb11